solidity/contracts/converter/BancorConverter.sol

```diff
@@ -183,6 +178,18 @@ contract BancorConverter is IBancorConverter, SmartTokenController, ContractRegi
     function withdrawETH(address _to) public ownerOnly only(BANCOR_CONVERTER_UPGRADER) {
         require(hasETHReserve());
         _to.transfer(address(this).balance);
+
+        // sync the ETH reserve balance	
+        syncReserveBalance(IERC20Token(0));

```

Comment:
May as well just do `reserveBalances[address(0)] = address(this).balance;`.

Reply:
Since this function isn't gas sensitive, I prefer code reuse on adding more custom code here